auto_subtitle/cli.py

```diff
@@ -18,6 +18,8 @@ def main():
                         default=".", help="directory to save the outputs")
     parser.add_argument("--output_srt", type=str2bool, default=False,
                         help="whether to output the .srt file along with the video files")
+    parser.add_argument("--output_txt", type=str2bool, default=False,
+                        help="whether to save the subtitles in a txt file")
```

Comment:
Maybe this would read better?
```suggestion
                        help="whether to also output the subtitles as a .txt file")
```
.....the existing description implies that a `.txt` file is generated instead of other outputs, which isn't the case I note from reading the code.